src/components/key_pad_menu/key_pad_menu_item.tsx

```diff
@@ -238,7 +245,7 @@ export const EuiKeyPadMenuItem: FunctionComponent<EuiKeyPadMenuItemProps> = ({
 
   const renderContent = () => (
     <span className="euiKeyPadMenuItem__inner">
-      {checkable ? renderCheckableElement() : renderBetaBadge()}
+      {checkable && renderCheckableElement()}
```

Comment:
The `checkable` check is already in `renderCheckableElement()`

```suggestion
      {renderCheckableElement()}
```